### PR TITLE
Make splitting job and test files optional

### PR DESCRIPTION
**Pull request by jmchilton**
I think that makes writing tests easier and I think @bgruening agrees (https://github.com/galaxyproject/training-material/pull/2021#discussion_r464233812)

### CODE REVIEW

File: planemo/commands/cmd_workflow_test_init.py [hunk 1]
```diff
@@ -18,29 +18,35 @@
...
          'doc': 'Test outline for %s' % path_basename,
-         'job': os.path.basename(job_output),
-         'outputs': output_stubs_for_workflow(workflow_path)
+         'job': job,
+         'outputs': output_stubs_for_workflow(workflow_path),
     }]
+    if split_test:
+        job_output = new_workflow_associated_path(workflow_path, suffix="job1")
+        if not can_write_to_path(job_output, **kwds):
+            ctx.exit(1)
```
Comment (mvdbeek) [inline planemo/commands/cmd_workflow_test_init.py:46]: Does that print some information about why this failed ?

Comment (jmchilton) [reply planemo/commands/cmd_workflow_test_init.py:46]: I think so:

``error("%s already exists, exiting." % path)``

Comment (jmchilton) [reply planemo/commands/cmd_workflow_test_init.py:46]: It'd be good to also check the directory exists and is writable... but yeah...

**Comment by bgruening** [comment]
Indeed, I think as the default behavior this is better. For power users, the split one is probably best. I like it! Thanks both!

**Pull request merged**